guides/common/modules/proc_troubleshooting-permission-issues.adoc

```diff
@@ -0,0 +1,26 @@
+[id="troubleshooting-permission-issues"]
+= Troubleshooting permission issues
+
+You can perform a health check on {ProjectServer} to find permission issues.
+
+.Procedure
+. On your {ProjectServer}, find duplicate permissions:
```

Comment:
Lines 7 and 21 seem to inconsistently describe what the command does. On line 7, you write that the purpose of the command is to "find duplicate permissions"; on line 21, it's "missing or duplicate permissions". Can you try to describe the command in both places the same way?